src/rpc/types/mod.rs

```diff
@@ -260,7 +260,7 @@ pub struct ApiState {
 
 lotus_json_with_self!(ApiState);
 
-#[derive(Clone, Serialize, Deserialize, PartialEq, Eq, JsonSchema)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "PascalCase")]
 pub struct SectorOnChainInfo {
```

Comment:
I assume there's no way of not re-defining this?

Reply:
I think so, this is the struct for LotusJson serde.